graphitepager/graphite_data_record.py

```diff
@@ -8,7 +8,12 @@ class NoDataError(ValueError):
 class GraphiteDataRecord(object):
 
     def __init__(self, metric_string):
-        meta, data = metric_string.split('|')
+        meta, data = None, None
+        try:
+            meta, data = metric_string.rsplit('|',1)
+        except ValueError as e:
+            print "ValueError trying to split on '|': %s" % metric_string
+            raise e
```

Comment:
But even so, if we still get a ValueError we will report details for debugging here.